doc/examples/denoise_patch2self.py

```diff
@@ -130,6 +142,22 @@
 
 print("Entire denoised data saved in denoised_patch2self.nii.gz")
 
+"""
+Lastly, one can also use Patch2Self in batches if the number of gradient
+directions is very high (>=200 volumes). For instance, if the data has 300
+volumes, one can split the data into 2 batches, (150 directions each) and still
+get the same denoising performance. One can simply run Patch2Self using:
+`denoised_batch1 = patch2self(data[..., :150], bvals[:150])`
+`denoised_batch2 = patch2self(data[..., 150:], bvals[150:])`
+
+After doing this, the 2 denoised batches can be merged as follows:
+`denoised_p2s = np.concatenate((denoised_batch1, denoised_batch2), axis=3)`
+
+If using Patch2Self for research purposes, acquisition design tests, etc., one
```

Comment:
```suggestion
If using Patch2Self for research purposes, acquisition design tests, etc., one
```
I am not sure what the "research purposes" means here. The following "acquisition design, tests, etc." suggests that this is not something that users should do if they want to analyze the data? But I am not sure.

Reply:
Yes! Agreed... Taking that part of the line off. I just wanted to encourage people to experiment with the algorithm.